portal-web/docroot/html/taglib/ui/organization_search_container_results/page.jsp

```diff
@@ -20,12 +20,14 @@
 OrganizationSearchTerms searchTerms = (OrganizationSearchTerms)request.getAttribute("liferay-ui:organization-search-container-results:searchTerms");
 LinkedHashMap<String, Object> organizationParams = (LinkedHashMap<String, Object>)request.getAttribute("liferay-ui:organization-search-container-results:organizationParams");
 long parentOrganizationId = GetterUtil.getLong(request.getAttribute("liferay-ui:organization-search-container-results:parentOrganizationId"));
+
+Indexer<?> indexer = IndexerRegistryUtil.nullSafeGetIndexer(Organization.class);
```

Comment:
There isn't enough of a difference to matter from a performance perspective.  We use null safe get indexer in most cases, one will log a message the other can throw an NPE if the indexer is somehow not registered - of course an NPE won't ever happen if these indexers are registered. I used nullSafeGetIndexer because that's the one we use the most in the portal (but only slightly).